### PR TITLE
batch upload analysis files

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.14.1"}
-PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "add_list_cloud_blobs"}
+PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "inc-page-count"}
```
Comment (as2388) [inline Pipfile]: v0.0.8 is all you need to upgrade to for batch uploads. I think the page count bug fix can wait, but if you need it now feel free to release a new version of pipeline infrastructure.

File: automated_analysis.py [hunk 2]
```diff
@@ -621,35 +621,36 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         paths_to_upload = glob(f"{output_dir}/*.csv")
         for i, path in enumerate(paths_to_upload):
             log.info(f"Uploading CSV {i + 1}/{len(paths_to_upload)}: {path}...")
-            drive_client_wrapper.update_or_create(
-                path, pipeline_configuration.drive_upload.automated_analysis_dir, target_folder_is_shared_with_me=True
+            drive_client_wrapper.update_or_create_batch(
+                path, pipeline_configuration.drive_upload.automated_analysis_dir, target_folder_is_shared_with_me=True,
```
Comment (as2388) [inline automated_analysis.py]: This isn't a batch upload if you're only passing in a single path.

**Review by as2388 — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (IsaackMwenda) [reply Pipfile]: released and updated to v0.0.9

File: automated_analysis.py (hunk 2 above)
Comment (IsaackMwenda) [reply automated_analysis.py]: updated thanks.

**Review by as2388 — APPROVED** [review]

**Pull request merged**